http/api_handler.go

```diff
@@ -229,6 +234,7 @@ var apiLinks = map[string]interface{}{
 	"telegrafs": "/api/v2/telegrafs",
 	"users":     "/api/v2/users",
 	"write":     "/api/v2/write",
+	"delete":    "/api/v2/delete",
```

Comment:
I like this idea initially, 
then i realized we already have an endpoint for DELETE /api/v2/buckets/{id}, which will completely delete this bucket.
The difference is the predicate delete will only delete the time series of the bucket.

I wonder if this could trigger some confusion?
 
what do you think? @russorat , @ebb-tide , @desa , @jademcgough

Reply:
@goller using that logic, wouldn't `write` also be under a bucket id? i'm not qualified to comment on this but my opinion is that using the name of an HTTP method in the url may cause confusion for people.

Reply:
@russorat I just spoke to @kelwang and @jademcgough about this.  I can restate my primary concerns now... basically, this endpoint has another predicate system.

Perhaps it would be better to have only one predicate system like flux?

Perhaps this delete logic should actually be placed in /api/v2/query depending on the long term vision?